a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -164,7 +164,9 @@ def execute(self, vthunder, loadbalancer, added_ports, ifnum_master=None, ifnum_
 
         if not lb_exists_flag:
             added_ports[amphora_id] = []
-            added_ports[amphora_id].append(nics[1])
+            for nic in nics:
+                if nic.network_id == loadbalancer.vip.network_id:
+                    added_ports[amphora_id].append(nic)
```

Comment:
As this condition will work only for 1st loadbalancer, I think evenif nic/nics are appended in `added_ports[amphora_id]` , we are injecting **None** to "added_ports" in the `EnableInterface` tasks which are getting executed for 1st loadbalancer.
https://github.com/a10networks/a10-octavia/blob/master/a10_octavia/controller/worker/flows/a10_load_balancer_flows.py#L362

https://github.com/a10networks/a10-octavia/blob/master/a10_octavia/controller/worker/flows/a10_load_balancer_flows.py#L480